src/vue/backend/src/styles/formContentStyles.scss

```diff
@@ -1,9 +1,19 @@
 @import '~bootstrap/scss/forms';
 .form-content {
-    display: flex;
```

Comment:
In an upcoming PR, there are major updates to this file to make it correspond with the design. Be aware if this is merged in first that there will be merge conflicts.